plugins/modules/scaleway_organization_info.py

```diff
@@ -44,7 +44,7 @@
 scaleway_organization_info:
   description: Response from Scaleway API
   returned: success
-  type: complex
+  type: list
```

Comment:
```suggestion
  type: list
  elements: dict
```
Always add `elements` to `type: list` :)

Reply:
I wasn't sure about that, but yeah, I think it is likely so.

Reply:
If it's not, then the sample is wrong :)